scripts/random-responses.js

```diff
@@ -123,6 +123,12 @@ module.exports = robot => {
     configs.forEach(async config => {
       attachTrigger(robot, config);
     });
+
+    robot.hear(/^fact of facts$/i, async res => {
+      responseFrom(robot, configs[Math.floor(Math.random() * configs.length)])(
```

Comment:
I'm pretty sure I see what you're doing here, but I think a comment for the next person reading this might help.